app/controllers/sessions_controller.rb

```diff
@@ -17,6 +17,14 @@ def hoge
     g = 1 unless c
   end
 
+
+  def hoge2
+    var = "aaaaa"  
+    var = "bbbbb"  
+    var2 = "ccccc"  
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/UselessAssignment: Useless assignment to variable - var2. Did you mean var?